styles/globals.css

```diff
@@ -167,3 +167,19 @@ code {
 .text-red-600 {
   color: #dc2626;
 }
+
+.git-link {
```

Comment:
@aanchal-fatwani Great job using the tailwind. But are we using this? If not, please cleanup and let me know. We are near to merge

Reply:
Could you please recheck this change? I had cleaned it up with the last commit.

Reply:
Done